server_test.go

```diff
@@ -154,7 +153,7 @@ func assertMetric(t *testing.T, metrics DDMetricsRequest, metricName string, val
 
 // setupVeneurServer creates a local server from the specified config
 // and starts listening for requests. It returns the server for inspection.
-func setupVeneurServer(t *testing.T, config Config, transport http.RoundTripper) *Server {
+func setupVeneurServer(t testing.TB, config Config, transport http.RoundTripper, mSink metricSink, sSink spanSink) *Server {
```

Comment:
Can you document what happens if no other sink is provided (and what the blackhole sink means, in the context of a test?)